src/signals/IncidentMap/components/IncidentMap/styled.ts

```diff
@@ -20,14 +20,22 @@ export const Wrapper = styled.div`
   box-sizing: border-box;
   display: flex;
   flex-direction: row-reverse;
+  flex: 0 0 67%;
+  overflow-y: auto;
 
   @media screen and ${breakpoint('max-width', 'tabletM')} {
     flex-direction: column;
+    flex: 0 0 50%;
   }
 `
+export const MapWrapper = styled.div`
+  height: 100%;
+  width: 100%;
+  z-index: 0;
+`
 
 export const StyledMap = styled(Map)`
   height: 100%;
   width: 100%;
-  z-index: 0;
+  z-index: 1;
 `
```

Comment:
It is not necessary to change any styling of the incidentMap. The GPS Button should work independently. Double checked, and you can remove all the added styling here if you also remove the `z-index: 1` from the GPS styling

Reply:
Done, I should pay more attention to removing code that was part of an attempt that went nowhere.